mmv1/products/networkconnectivity/Group.yaml

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Group
+legacy_long_form_project: true
+base_url: projects/{{project}}/locations/global/hubs/{{hub}}/groups
+create_url: projects/{{project}}/locations/global/hubs/{{hub}}/groups/{{name}}?updateMask=autoAccept.autoAcceptProjects
+self_link: projects/{{project}}/locations/global/hubs/{{hub}}/groups/{{name}}
+create_verb: 'PATCH'
+update_verb: 'PATCH'
+update_mask: true
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/network-connectivity/docs/network-connectivity-center/concepts/overview'
+  api: 'https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1beta/projects.locations.global.hubs.groups'
+async:
+  type: 'OpAsync'
+  operation:
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result:
+    path: 'targetLink'
+  error:
+    path: 'error/errors'
+    message: 'message'
+examples:
+  - name: 'network_connectivity_group_basic'
+    primary_resource_id: 'primary'
+    vars:
+      hub_name: "hub1"
+      auto_accept_project_1_name: "foo"
+      auto_accept_project_2_name: "bar"
+description: The NetworkConnectivity Group resource
+parameters:
+  - name: 'hub'
+    type: ResourceRef
+    resource: 'Hub'
+    imports: 'id'
+    description: |
+      The name of the hub. Hub names must be unique. They use the following form: projects/{projectNumber}/locations/global/hubs/{hubId}
+    immutable: true
+    required: true
+    url_param_only: true
+    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+properties:
+  - name: name
+    type: Enum
+    description: 'The name of the group. Group names must be unique. They use the following form: `projects/{projectNumber}/locations/global/hubs/{hub}/groups/{groupId}`'
+    diff_suppress_func: 'tpgresource.CompareSelfLinkOrResourceName'
+    required: true
+    immutable: true
+    custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.tmpl'
+    enum_values:
```

Comment:
Can you please add the field `exclude_sweeper` and the comment  to the resource?

```
# A handwritten sweeper is needed as the resource name can only be `default`, `center`, and `edge`.
exclude_sweeper: true
```

Example https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/accesscontextmanager/AccessLevel.yaml#L66